### DISCUSSION TITLE
TypeError: Cannot read properties of undefined .split()

### DISCUSSION THREAD

**Discussion by shaffy9633** [WalletConnect/walletconnect-monorepo#3176] [Developer Support]
Hi I have been getting the following error showing up on the web3Modal UI. The error would show up on the area of where the arrow is pointed at
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/110626159/be8bd116-59de-4745-8cb0-5ec8a61b2412)

**Comment by glitch-txs** [comment]
hi, do you have any public repo we can test and reproduce this issue?
